src/Compilers/Core/Portable/Compilation/IExpression.cs

```diff
@@ -841,10 +841,36 @@ public interface IConversionExpression : IHasOperatorMethodExpression
         /// Kind of conversion.
         /// </summary>
         ConversionKind ConversionKind { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an identity conversion.
+        /// </summary>
+        bool IsIdentity { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit ('widening' in VB) conversion.
+        /// </summary>
+        bool IsImplicit { get; }
+
         /// <summary>
         /// True if and only if the conversion is indicated explicity by a cast operation in the source code.
         /// </summary>
         bool IsExplicit { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit numeric conversion or explicit numeric conversion. 
+        /// </summary>
+        bool IsNumeric { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit nullable conversion or explicit nullable conversion.
+        /// </summary>
+        bool IsNullable { get; }
+
+        /// <summary>
+        /// Returns true if the conversion is an implicit reference conversion or explicit reference conversion.
+        /// </summary>
+        bool IsReference { get; }
```

Comment:
> Alternatively, we can add one new property that returns a different new interface with all these properties

I have gone ahead and made that change.  i think it does make things much nicer.  It also makes things more uniform in the API wrt to the ```Conversion``` types.  i.e. the existing ```Conversion``` types *are* implementations of the new ```IConversion``` interface.  So things feel very uniform if you're using the semantic model API or if you're using the IOperation API.  The major difference is that IOperation just exposes less info (which is how IOperation normally works).